### ISSUE TITLE
xcode on real device

### ISSUE THREAD

**Issue by itsoulos** [foxlet/macOS-Simple-KVM#391]
Hello,
can I run xcode on a real device (iphone or ipad) using this approach?

**Comment by cjoshmartin** [comment]
@itsoulos yeah probably just pass through the port the device lives on. 

https://github.com/foxlet/macOS-Simple-KVM/blob/master/docs/guide-passthrough.md

https://youtu.be/QWZ_LjzT39k?t=530

**Comment by walker84837** [comment]
@itsoulos Yeah, sure. Just make sure the device you're using has enough storage and RAM to run a VM.
Since it's 2023 now, the most recent version is macOS Ventura, which means you're gonna need to update macOS to the latest version, but updating macOS by using System Settings isn't going to work (See #436). From my tests, editors like VSCode run perfectly without having passed-through a GPU, so I don't think you need to pass-through a GPU if you don't have one that isn't integrated.
